### ISSUE TITLE
Accessibility: MAS05A: Keyboard: Image does not have any visible keyboard focus.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#41]
#A11yMAS
#MAS05A
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in Edge.
2. Navigate to the "Extensible and customizable" text in middle of the page.
3. Try to access the image beside the text.

**Actual Result:**
Unable to access the image while navigating through keyboard.

**Expected Result:**
The user should be able  to access the all the interactive elements available in the page.

**User Impact:**
Users with visual and/or motor impairments who rely on keyboard may find it difficult while navigating with the tab to use the Application. 

**Screenshots:**
![inspect behaviour of the image](https://cloud.githubusercontent.com/assets/24825441/21713294/394f2fda-d41f-11e6-9979-2505a5110be2.png)
![unable to access the image](https://cloud.githubusercontent.com/assets/24825441/21713293/394f05b4-d41f-11e6-97c3-dd5810023c8e.png)

**Comment by chrisdias** [comment]
I can tab to the image and pressing Enter opens the marketplace.
